new_project/README.md

```diff
@@ -4,7 +4,7 @@ This project was bootstrapped with [Create Comit App](https://github.com/comit-n
 
 In the project directory, you can run:
 
-### `yarn create comit-app start-env`
+### `yarn env`
```

Comment:
Minor nit:

If we want the command to be `env`, I'd _document_ it as `yarn run env`. If we don't want to use `yarn run`, I'd rename the command to `yarn start-env`.

I am leaning towards `start-env` because it has a verb in the actual command and does not abuse the `run` sub command to make it sound like a command.

When reading `yarn env`, I'd expect it to _print_ a list of environment variables or something.